docs/resources/elb_monitor.md

```diff
@@ -27,25 +31,48 @@ The following arguments are supported:
 * `region` - (Optional, String, ForceNew) The region in which to create the ELB monitor resource. If omitted, the
   provider-level region will be used. Changing this creates a new monitor.
 
-* `pool_id` - (Required, String, ForceNew) The id of the pool that this monitor will be assigned to.
+* `pool_id` - (Required, String, ForceNew) Specifies the ID of the backend server group for which the health check is
+  configured.
```

Comment:
please add `Changing this ...` as this parameter is **ForceNew**.

Reply:
done